bitnami/keycloak/ci/ct-values.yaml

```diff
@@ -0,0 +1,2 @@
+service:
+  type: ClusterIP
```

Comment:
What happens with other available YAMLs in Keycloak? I mean:
- values-ha.yaml
- values-hpa-pdb.yaml
- values-metrics-and-ingress.yaml
Same question applies to the rest of charts

Reply:
We thought on renaming all the `ci/*.yaml` files to `ci/*-values.yaml`. However, it actually doesn't make sense. Many of the files are trying to test production settings and those can't be applied in a kind cluster.

Reply:
It makes sense